test_team_1.py

```diff
@@ -89,6 +89,42 @@ def test_iban_code(self):
 
     def test_ip_address(self):
         """Test IP_ADDRESS functionality"""
+        net1s = ['168', '192', '255'] #First part of the IP Address
+        net2 = ['243', '542', '495']  #Second part of the IP Address
+        net3 = ['1']  #Third part of the IP Address
+        hostID = ['1', '32', '170'] #Last part of IP and the host ID
+        
+        IP61 = ['2001:0db8:85a3:','09C0:876A:130B:'] #First part of the IP
+        IP62 = ['8a2e:0370:7334','2001:0db8:85a3'] # Second part of the IP
+        
+        IPAddres = []
+        IP6 = []
+        for n1 in net1s:
+            for n2 in net2:
+                for n3 in net3:
+                    for h in hostID:
+                        IPAddres = '.'.join([n1,n2,n3,h])
+                        result = analyze_text('IP_ADDRESS is ' + IPAddres, ['IP_ADDRESS']) #One with Context words
+                        result2 = analyze_text('my Address is ' + IPAddres, ['IP_ADDRESS'])#One without context words
+                        print('\n',IPAddres)
+                        print(result, '\n')
+                        print(result2)
+                        break
```

Comment:
You are not actually testing anyting here, only printing the results.  You need an appropriate self.assert() statement to actually test something.